copilot/fastapi/app/config.py

```
@@ -0,0 +1,91 @@
+    Retrieve the Kubernetes service account token.
+    
+    This function reads the token from the file specified by kubernetes_token_path and returns
+    it as a stripped string. If the token cannot be read, it raises an HTTPException
+    with a 500 status code.
+    """
+    settings = get_settings()
+    try:
+        with open(settings.kubernetes_token_path, "r") as f:
+            return f.read().strip()
+    except OSError as e:
+        raise HTTPException(status_code=500, detail=f"Unable to read kubernetes token: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)